assignment1/styles.css

```diff
@@ -0,0 +1,58 @@
+body {
+  padding: 0;
+  margin: 0;
+  font-family: sans-serif;
+  background-color: #5cb4f0;
+}
+.name {
+  margin-top: 50px;
+  margin-bottom: 10px;
+}
+.form {
+  padding: 30px;
+  background-color: white;
+  /*height: 550px;*/
+  width:50%;
+  border-radius: 2%;
+  text-align: center;
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  /*-ms-transform: translate(-50%, -50%);*/
+  transform: translate(-50%, -50%);
+}
+.symbol {
+  display: inline-block;
+  padding-right: 20px;
+  padding-top: 20px;
+  margin-bottom: -5px;
+  
+}
+.data{
+  width:50%;

```

Comment:
merge properties like margin, padding, border into a single line for a single element